tests/integration/research/utils/test_int_simulator.py

```diff
@@ -0,0 +1,205 @@
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from footbot.data.utils import run_query
+from footbot.data.utils import set_up_bigquery
+from footbot.data.utils import write_to_table
+from footbot.research.utils.simulator import get_elements_df
+from footbot.research.utils.simulator import get_results_df
+from footbot.research.utils.simulator import make_team_selection
+from footbot.research.utils.simulator import make_transfers
+from footbot.research.utils.simulator import simulate_event
+from footbot.research.utils.simulator import simulate_events
+
+
+@pytest.fixture
```

Comment:
since the client isn't stateful and probably take a non-zero amount of time to instantiate, i would mark this as `pytest.fixture(scope='session')` - this will just instantiate a single instance of it for all tests instead of creating a new client for each test